doc-Installing_on_AWS/topics/installation.adoc

```diff
@@ -169,30 +195,37 @@ To import the appliance:
 . Create a containers.json file:
 +
 ----
-[
-  {
-  "Description": "NAME OF INSTANCE",
-  "Format": "vhd",
-  "UserBucket": {
-        "S3Bucket": "BUCKET WITH UPLOADED .VHD IMAGE",
-        "S3Key": "NAME OF .VHD IMAGE"
-      }
-  }
-]
+{
+    "Description": " NAME OF IMPORTED SNAPSHOT IN AWS",
+    "Format": "vhd",
+    "UserBucket": {
+        "S3Bucket": "BUCKET WITH UPLOADED .VHD SNAPSHOT",
+        "S3Key": "PATH OF .VHD SNAPSHOT"
+    }
+}
 ----
 +
+See the link://docs.aws.amazon.com[AWS documentation] on VM import and export requirements,
+such as image formats, instances, volume and file system types, and using regions.
++
 . Use the AWS-CLI tools to import a disk as a snapshot. See the
 link://docs.aws.amazon.com[AWS documentation] on using VM Import/Export to
 import a disk as a snapshot.
 +
+[NOTE]
+====
+You can either specify a region, or if not, ensure that the S3 bucket is in the same
+region where you want to import the snapshot.
+====
++
 ----
-$ aws ec2 import-snapshot --disk-containers file://containers.json
+$ aws ec2 import-snapshot --disk-container file://containers.json
 ----
 +
 . Check the progress of your snapshot import by running the following command:
 +
 ----
- $ aws ec2 describe-import-snapshot-tasks --import-task-ids import-ami-AMI_ID_GOT_FROM_RESPONSE
+ $ aws ec2 describe-import-snapshot-tasks --import-task-ids snap-SNAPSHOT_ID_GOT_FROM_RESPONSE
```

Comment:
import-snap-SNAPSHOT_ID_GOT_FROM_RESPONSE or just SNAPSHOT_ID_GOT_FROM_RESPONSE

Reply:
Changed to second option: "SNAPSHOT_ID_GOT_FROM_RESPONSE"